### DISCUSSION TITLE
Stack spacing not working

### DISCUSSION THREAD

**Discussion by bcarusodev** [chakra-ui/chakra-ui#7759] [Q&A]
Hi there,

My Stack component doesn't apply `spacing` prop, i think it could be a dependency version problem (but idk)

![image](https://github.com/chakra-ui/chakra-ui/assets/29601669/ff09a4a4-fb35-4173-a8b3-1539b2be2446)

I have this Stack with 2 components inside, in a past i can use spacing but when i used yarn install something updated and broke the `spacing` prop!

I have the `ChakraProvider` properly set on `<App/>` i'm using NextJS

Anyone can help me? Thanks!

**Comment by bcarusodev** [comment]
I think is a yarn lockfile problem on a dependency because i used a old lockfile and it works normally but if i create the lockfile from `yarn install` command the bug appears

**Reply by artfauq** [reply]
Hey @silvergraphs, did you manage to solve your issue ? I think I'm having the same problem
